Makefile

```diff
@@ -32,6 +32,11 @@ init:                           ## Installs tools to $GOPATH/bin (which is expec
 gen:                            ## Generate files.
 	go generate ./...
 
+gen-init:
```

Comment:
what does this command do?

Reply:
It generates initial model files (like models.go) that should be edited by hands.
We use only a few tables from performance schema, but it is likely that we will use more in the near future, that's why I kept it for now.

Reply:
Could you add a comment to makefile? Like `## Generate initial model files`

Reply:
That comment will make it a public target – visible in `make` output. That target is intended to be private.